src/vsr/journal.zig

```diff
@@ -278,7 +279,7 @@ pub fn Journal(comptime Replica: type, comptime Storage: type) type {
             return self.entry_for_op(header.op + 1);
         }
 
-        pub fn next_offset(self: *Self, header: *const Header) u64 {
+        pub fn next_offset(_: *Self, header: *const Header) u64 {
```

Comment:
Maybe we should just remove the first argument instead?

Reply:
Done